pkg/core/native/notary.go

```diff
@@ -278,10 +278,7 @@ func (n *Notary) withdraw(ic *interop.Context, args []stackitem.Item) stackitem.
 	if !ok {
 		return stackitem.NewBool(false)
 	}
-	to := from
-	if !args[1].Equals(stackitem.Null{}) {
-		to = toUint160(args[1])
-	}
+	to := toUint160(args[1])
```

Comment:
That's wrong and I think it'll be fixed in C#.

Reply:
I hope so, but until that let's keep this commit in a separate PR.